avocado-setup.py

```diff
@@ -334,6 +349,14 @@ def bootstrap(enable_kvm=False):
                   err_str="Failed to create test repo dir. Error: ")
     for repo in TEST_REPOS:
         get_repo(repo, TEST_DIR)
+    ffdc_bootstrap()
+
+
+def ffdc_bootstrap():
+    logger.info("Bootstrapping ffdc")
+    helper.runcmd('mkdir -p %s' % FFDC_DIR,
+                  debug_str="Creating FFDC repo dir %s" % FFDC_DIR)
+    get_repo(FFDC_REPO, FFDC_DIR, True)
```

Comment:
Can we have this similar to the existing way, instead of doing it in a function ?
Or, if possible, change the existing way also to use a single template function instead.

Reply:
ok, I am fine with both the approach, I feel move Test also in different Function will be good what you say it is easy to maintain on a longer run, what you say?

Reply:
@PraveenPenguin here, we are doing 3 things:
log
create dir
call get_repo

I think first 2 things can also be done by get_repo itself, based on additional params (like create_dir=True or directory=<dir name>, something like that)

Reply:
sure will do that